packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -126,7 +126,7 @@ export class VaButton {
       <Host>
         <button
           aria-disabled={ariaDisabled}
-          aria-label={label || buttonText}
+          aria-label={label}
```

Comment:
Can we add a test to show that the `label` prop results in an `aria-label` on the button?